packages/amplify_authenticator/lib/src/theme/amplify_theme.dart

```diff
@@ -262,103 +262,94 @@ class AmplifyColors {
 class AmplifyTextTheme extends TextTheme {
   const AmplifyTextTheme()
       : super(
-          headline1: h1,
-          headline2: h2,
-          headline3: h3,
-          headline4: h4,
-          headline5: h5,
-          headline6: h6,
-          subtitle1: bodyLarge,
-          bodyText2: body,
-          caption: bodySmall,
-          button: button$,
+          headline1: _h1,
+          headline2: _h2,
+          headline3: _h3,
+          headline4: _h4,
+          headline5: _h5,
+          headline6: _h6,
+          subtitle1: _bodyLarge,
+          bodyText2: _body,
+          caption: _bodySmall,
+          button: _button$,
         );
 
   static const fontPackage = 'amplify_authenticator';
   static const fontFamily = 'Inter';
 
-  static const h1 = TextStyle(
+  static const _h1 = TextStyle(
     fontFamily: fontFamily,
     package: fontPackage,
     fontWeight: FontWeight.bold,
     fontSize: 64,
     height: 72 / 64,
   );
 
-  static const h2 = TextStyle(
+  static const _h2 = TextStyle(
     fontFamily: fontFamily,
     package: fontPackage,
     fontWeight: FontWeight.bold,
     fontSize: 56,
     height: 64 / 56,
   );
 
-  static const h3 = TextStyle(
+  static const _h3 = TextStyle(
     fontFamily: fontFamily,
     package: fontPackage,
     fontWeight: FontWeight.bold,
     fontSize: 48,
     height: 56 / 48,
   );
 
-  static const h4 = TextStyle(
+  static const _h4 = TextStyle(
     fontFamily: fontFamily,
     package: fontPackage,
     fontWeight: FontWeight.bold,
     fontSize: 40,
     height: 48 / 40,
   );
 
-  static const h5 = TextStyle(
+  static const _h5 = TextStyle(
     fontFamily: fontFamily,
     package: fontPackage,
     fontWeight: FontWeight.bold,
     fontSize: 32,
     height: 40 / 32,
   );
 
-  static const h6 = TextStyle(
+  static const _h6 = TextStyle(
     fontFamily: fontFamily,
     package: fontPackage,
     fontWeight: FontWeight.bold,
     fontSize: 24,
     height: 32 / 24,
   );
 
-  static const bodyLarge = TextStyle(
+  static const _bodyLarge = TextStyle(
```

Comment:
A property on TextTheme was renamed to bodyLarge, which made this conflict. I made all these private just on the off chance this happens again.